.github/workflows/test.yml

```diff
@@ -40,3 +48,123 @@ jobs:
         run: mix test
         env:
           MIX_ENV: test
+        
+  integration_test:
```

Comment:
I am not sure why this is named `integration_test` as it executes the code formatter checks in each variant 🤔  Maybe naming it `test_variant` would be more descriptive? Also in that case, since both `unit_test` and `test_variant` are independent, it could be moved to another workflow file entirely -> `test_variant.yml`? It would make this file easier to manage in the long run IMO.

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/24/commits/cf66116acbae3333aa7ce8de2fe068d41c51a19f